docs/_includes/layouts/ethics.njk

```diff
@@ -0,0 +1,69 @@
+{% extends "./base.njk" %}
+
+{%- from "node_modules/@ministryofjustice/frontend/moj/components/side-navigation/macro.njk" import mojSideNavigation %}
+
+{% macro appSideNavigation(config) %}
+  {% set config = config|addActiveAttribute(page.filePathStem | url) %}
+
+  {{ mojSideNavigation(config) }}
+{% endmacro %}
+
+{% block content %}
+
+<div class="govuk-width-container">
+
+  <div class="govuk-phase-banner">
+    <p class="govuk-phase-banner__content">
+      <strong class="govuk-tag govuk-phase-banner__content__tag">
+        Beta
+      </strong>
+
+      <span class="govuk-phase-banner__text">
+      This is new – your <a class="govuk-link" href="mailto:design-system@digital.justice.gov.uk">feedback</a> will help us to improve it.
+      </span>
+
+    </p>
+  </div>
+
+  <div class="govuk-width-container govuk-grid-row app-flex-container">
+    <div class="govuk-grid-column-one-third">
+      {{ appSideNavigation({
+        classes: 'govuk-!-padding-top-6',
+        sections: [
+          {
+            heading: {
+              text: "Ethics"
+            },
+            items: [
+              {
+                text: 'Principles',
+                href: ('/ethics/principles' | url)
+              },
+              {
+                text: 'Standards',
+                href: ('/ethics/standards' | url)
+              },
+              {
+                text: 'Do an ethics assessment',
```

Comment:
I reckon either the other links should also match the page title, or this one should be a contraction too

Reply:
Changed to 'Ethics assessment'.